tools/tojs/cocos2dx_studio.ini

```diff
@@ -56,7 +56,8 @@ skip = *::[createInstance ^visit$ copyWith.* onEnter.* onExit.* ^description$ ge
         ColliderBody::[getContourData getCalculatedVertexList],
         SceneReader::[*],
         CSLoader::[*],
-        ActionTimelineCache::[*]
+        ActionTimelineCache::[*],
+        Frame::[(s|g)etEasingParams]
```

Comment:
This should be bound in manual bindings, Cocos Studio parser will need these functions